packages/devtools_app/lib/src/call_stack.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import 'common_widgets.dart';
+import 'utils.dart';
+
+/// The type of data provider function used by the CopyToClipboard Control.
```

Comment:
All of this code is already defined in console.dart. Instead of copying here, please delete this file and move the code from console.dart into common_widgets.dart.